scss/_button-group.scss

```diff
@@ -39,7 +39,7 @@
   // Prevent double borders when buttons are next to each other
   > .btn:not(:first-child),
   > .btn-group:not(:first-child) {
-    margin-left: -$btn-border-width;
+    margin-left: $btn-group-not-first-child-margin-left;
```

Comment:
This would never change really, so it doesn't make sense to have a unique variable for it.